site/en/docs/webstore/best_practices/index.md

```diff
@@ -45,24 +45,19 @@ standards of performance, security, and user experience, as summarized by the fo
 
 ## Support Google Accounts
 
-If your app requires user login, we recommend that you provide at least some support for Google
-Accounts. The reason is that if a user purchases your app from the Chrome Web Store, odds are the
-user is already logged into a Google Account whenever they use your app. Reducing the number of
+If your extension requires user login, we recommend that you provide at least some support for Google
+Accounts. Google Accounts can help you provide a better user experience, since users of the Chrome
+Web Store are likely to be logged in already, and they won't have to set up and remember yet another
+username and password. Reducing the number of
 logins improves the user's experience.
```

Comment:
```suggestion
If your extension requires user login, we recommend that you provide at least some support for
Google Accounts. Google Accounts can help you provide a better user experience; since users of the
Chrome Web Store are likely to be logged in already, they won't have to set up and remember yet
another username and password. Reducing the number of logins improves the user's experience.
```

Slight grammar tweeks. Rewrap at 100 characters.

Reply:
'since users of the ...' should be the start of a new sentence.